app/_features/event/components/event-form.tsx

```diff
@@ -0,0 +1,493 @@
+  const [eventName, setEventName] = useState('');
+  const [eventDescription, setEventDescription] = useState('');
+
+  const { navigateTo } = useNavigate();
+
+  const handleRemoveImage = useCallback(() => {
+    updateImageData({ type: 'Reset' });
+  }, []);
+
+  const handleFileSelect = useCallback(
+    (event: ChangeEvent<HTMLInputElement>) => {
+      const file = event.target.files && event.target.files[0];
+
+      if (file) {
+        updateImageData({ type: 'PickFile', payload: file });
+
+        document.dispatchEvent(new CustomEvent('open:image-cropper'));
+      }
+    },
+    []
+  );
+
+  useEffect(() => {
+    if (!user) document.dispatchEvent(new CustomEvent('open:sign-in-modal'));
+  }, [user]);
+
+  useEffect(() => {
+    if (eventName.trim().length === 0) {
+      setTitleValid(false);
+    } else {
+      setTitleValid(true);
+    }
+  }, [eventName]);
+
+  useEffect(() => {
+    if (eventDescription.trim().length === 0) {
+      setDescValid(false);
+    } else {
+      setDescValid(true);
+    }
+  }, [eventDescription]);
+
+  useEffect(() => {
+    if (startTime.hour > endTime.hour) {
+      setEndTime({
+        hour: `${startTime.hour}`,
+        minute: `${startTime.minute}`,
+      });
+    }
+  }, [startTime.hour, startTime.minute, endTime.hour]);
+
+  const createEvent = useCallback(
+    async (isDraft: boolean) => {
+      if (
+        eventDescription.trim().length === 0 ||
+        eventName.trim().length === 0
+      ) {
+        document.dispatchEvent(new CustomEvent(incompleteFieldEvent));
+        return;
+      }
+
+      const finalEndpoint = `/api/events/create`;
+
+      const eventStartTime = new Date(
+        date.setHours(parseInt(startTime.hour), parseInt(startTime.minute), 0)
+      );
+      const eventEndTime = new Date(
+        date.setHours(parseInt(endTime.hour), parseInt(endTime.minute), 0)
+      );
+
+      let imageFile: File | Blob | null = null;
+
+      if (imageData.imageBlob) {
+        imageFile = await compressImage(imageData.imageBlob, 280, 560, 0.8);
+      }
+
+      const data = JSON.stringify({
+        name: eventName,
+        startTime: eventStartTime.toISOString(),
+        endTime: eventEndTime.toISOString(),
+        description: eventDescription.replace(/(?:\r\n|\r|\n)/g, '<br>'),
+        host: user?.name,
+        isPublished: !isDraft,
+      });
+
+      const formData = new FormData();
+
+      if (imageFile) {
+        formData.append('image', imageFile, 'poster.webp');
+      }
+      formData.append('data', data);
+
+      const respEvent = await InternalApiFetcher.post(finalEndpoint, {
+        body: formData,
+        headers: undefined,
+      });
+
+      if (respEvent.ok) {
+        const redirectPath = new URL(
+          `/event/${respEvent.data.slug}`,
+          window.location.origin
+        ).href;
+        navigateTo(redirectPath);
+      } else {
+        console.log(respEvent.message);
+      }
```

Comment:
No need to log the message with else statement. Use alert instead if you want to notify the user.

Reply:
removed the console log and replaced it with an alert instead 
removed the unnecessary console log https://github.com/inlivedev/inlive-room/pull/130/commits/06de4972dc0b8b36258652c019bd4a9acd9245ff#diff-9e2bd548722a266fc9bcce2be48f3d5b8b48215f5adbedbfbffa138083b1e33eL193-R184